docs/platforms/javascript/common/configuration/integrations/browsertracing.mdx

```diff
@@ -38,3 +38,5 @@ Sentry.init({
   integrations: [Sentry.browserTracingIntegration()],
 });
 ```
+
+See <PlatformLink to="/tracing/instrumentation/automatic-instrumentation/#configuration-options">Configuration Options</PlatformLink> for a full list of availalable options for `browserTracingIntegration` 
```

Comment:
kinda awkward that this page doesn't show the options. We can also think about moving all the options to this page from the tracing docs if reviewers prefer this.